src/components/Markdown/ChatMDComponent.tsx

```diff
@@ -61,22 +61,22 @@ export const getMDComponents = (
     table({ children }) {
       return (
         <div className="max-w-full overflow-auto">
-          <table className="border-collapse border border-black px-3 py-1 text-sm dark:border-gray-100">
+          <table className="border-collapse border border-primary px-3 py-1 text-sm">
             {children}
           </table>
         </div>
       );
     },
     th({ children }) {
       return (
-        <th className="break-words border border-black bg-gray-500 px-3 py-1 text-sm text-gray-200 dark:border-gray-100">
+        <th className="break-words border border-primary bg-layer-4 px-3 py-1 text-sm text-secondary">
           {children}
         </th>
       );
     },
     td({ children }) {
       return (
-        <td className="break-words border border-black px-3 py-1 text-sm dark:border-gray-100">
+        <td className="break-words border border-primary px-3 py-1 text-sm">
```

Comment:
`border-primary` is much lighter than `border-black`. Maybe `border-tertiary` shoud be here and above?